test/parallel/test-uv-errmap.js

```diff
@@ -0,0 +1,24 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
```

Comment:
```suggestion
require('../common');
```
If this `common` is not being used, we can just require it like this to make the linter happy.